lightningd/opening_control.c

```diff
@@ -174,6 +174,10 @@ wallet_commit_channel(struct lightningd *ld,
 	/* old_remote_per_commit not valid yet, copy valid one. */
 	channel_info->old_remote_per_commit = channel_info->remote_per_commit;
 
+    // init new channel with global fee settings per default
+    u32 feerate_base = ld->config.fee_base;
```

Comment:
This fixes the feerate to the configured value at the time the channel was opened, which may not always be the desired effect. A user may restart its node with different fees, but all its channels are pinned to the config value before the change. Since you are checking for `0` you might pass in `0` here and default to the config value instead. Otherwise this needs documenting.

Reply:
Also if a channel defaults to `0` you should probably remove the arguments to `new_channel` again, since they're now fixed again.

Reply:
Both issues fixed by later commit.